src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
```

Comment:
If you have a class declaration and right after that you have a class' method, you should add an empty line between them. <br/>
But if you have a fields after class declaration, you don't have to add an empty line between them.
```suggestion
public class Lottery extends ColorSupplier {

    public Ball getRandomBall() {
```